website/docs/reference/commands/dbt-environment.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "About dbt environment command"
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. The `dbt environment` command is useful for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
+- Viewing your dbt Cloud configuration details (account ID, environment ID, environment name, connection type, and more).
+- CAN YOU MAKE CHANGES TO THE ENVIRONMENT?
```

Comment:
flagging this question

Reply:
Not currently, we plan to add this functionality at a later date. We shouldn't mention it yet.